runatlantis.io/docs/stats.md

```diff
@@ -9,3 +9,45 @@ Currently Statsd and Prometheus is supported. See configuration below for detail
 ## Configuration
 
 Metrics are configured through the [Server Side Config](server-side-repo-config.html#metrics).
+
+## Available Metrics
+
+Assuming metrics are exposed from the endpoint `/metrics` from the [metrics](server-side-repo-config.html#metrics) server side config e.g.
+```yaml
+metrics:
+  prometheus:
+    endpoint: "/metrics"
+```
+To see all the metrics exposed from atlantis service, make a GET request to the `/metrics` endpoint.
+```bash
+curl localhost:4141/metrics
+# HELP atlantis_cmd_autoplan_builder_execution_error atlantis_cmd_autoplan_builder_execution_error counter
+# TYPE atlantis_cmd_autoplan_builder_execution_error counter
+atlantis_cmd_autoplan_builder_execution_error 0
+# HELP atlantis_cmd_autoplan_builder_execution_success atlantis_cmd_autoplan_builder_execution_success counter
+# TYPE atlantis_cmd_autoplan_builder_execution_success counter
+atlantis_cmd_autoplan_builder_execution_success 10
+# HELP atlantis_cmd_autoplan_builder_execution_time atlantis_cmd_autoplan_builder_execution_time summary
+# TYPE atlantis_cmd_autoplan_builder_execution_time summary
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.5"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.75"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.95"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.99"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.999"} NaN
+atlantis_cmd_autoplan_builder_execution_time_sum 11.42403017
+atlantis_cmd_autoplan_builder_execution_time_count 10
+.....
+.....
+.....
+```
+Some of the important metrics which you would like to monitor are
+
+| Metric Name                                  | Metric Type                                                          | Purpose                                                                                                                                  |
+|----------------------------------------------|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| atlantis_cmd_autoplan_execution_error        | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | since starting of atlantis server, the number of times when [autoplan](autoplanning.html#autoplanning) on the MR’s has thrown error.     |
```

Comment:
We can shorten the description here by

- removing the period
- removing the mention "since starting" since this is true for all metrics
- removing the mention of MR since we have a lot of non gitlab users

```suggestion
| `atlantis_cmd_autoplan_execution_error`        | [counter](https://prometheus.io/docs/concepts/metric_types/#counter) | number of times when [autoplan](autoplanning.html#autoplanning) throws an error     |
```

We can also surround the metric name with backticks

Lets do this for each of the rows in this table

Reply:
updated as per the suggestion.